src/components/Coaching/CoachingList.tsx

```diff
@@ -78,3 +84,11 @@ export const CoachingList: React.FC<CoachingListProps> = ({
     </CoachingListWrapper>
   );
 };
+
+const LoadingCoach: React.FC = () => (
```

Comment:
Does this need the `span` wrapper? Also, can we move it above the main component like we do in most files?